org.eclipse.ice.commands/src/main/java/org/eclipse/ice/commands/ConnectionConfiguration.java

```diff
@@ -43,35 +58,104 @@ public ConnectionConfiguration() {
 	}
 
 	/**
-	 * Constructor which gives a particular hostname. Most useful for local command
-	 * execution since a username/password is unnecessary.
+	 * Setter function for
+	 * {@link org.eclipse.ice.commands.ConnectionConfiguration#username}
 	 * 
-	 * @param hname - hostname
+	 * @param uname
 	 */
-	public ConnectionConfiguration(String hname) {
-		hostname = hname;
+	public void setUsername(String uname) {
+		username = uname;
 	}
 
 	/**
-	 * Constructor which gives a particular username, password, and hostname
+	 * Getter for the username for a connection
 	 * 
-	 * @param uname - username for a connection
-	 * @param pwd   - password for the connection
-	 * @param hname - hostname for the connection
+	 * @return username
 	 */
-	public ConnectionConfiguration(String uname, String pwd, String hname) {
-		username = uname;
-		password = pwd;
+	public String getUsername() {
+		return username;
+	}
+
+	/**
+	 * Setter function for
+	 * {@link org.eclipse.ice.commands.ConnectionConfiguration#hostname}
+	 * 
+	 * @param hname
+	 */
+	public void setHostname(String hname) {
 		hostname = hname;
 	}
 
 	/**
 	 * Create getter and setter functions to access member variables
 	 */
-	
+
 	public String getHostname() {
 		return hostname;
-	
 	}
-	
+
+	/**
+	 * A getter to access
+	 * {@link org.eclipse.ice.commands.ConnectionConfiguration#name}
+	 * 
+	 * @return - name
+	 */
+	public String getName() {
+		return name;
+	}
+
+	/**
+	 * A setter to access
+	 * {@link org.eclipse.ice.commands.ConnectionConfiguration#name}
+	 * 
+	 * @return - name
+	 */
+	public void setName(String _name) {
+		name = _name;
```

Comment:
Inconsistent naming. Use "this.name = name"

Reply:
Okay, I formatted all the setter functions the same as you suggested.